recipes/echtvar/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "0.2.2" %}
+
+package:
+  name: echtvar
+  version: {{ version }}
+
+build:
+  number: 0
+  skip: true  # [osx]
+  run_exports:
+    - {{ pin_subpackage('echtvar', max_pin="x.x") }}
+
+source:
+  url: https://github.com/brentp/echtvar/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 6ca8070d6e5c13addc65064124c1a202321ce2a6779c462d5e849b8cc06abd7f
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('rust') }}
+    - conda-forge/label/rust_dev::rust_{{ target_platform }} 1.85.0
+    - conda-forge/label/rust_dev::rust 1.85.0.dev20241213
+    - {{ compiler('cxx') }}
+    - cargo-bundle-licenses
+    - clangdev
```

Comment:
```suggestion
```
Instead of setting `clangdev` here, you can just add these lines to `conda_build_config.yaml`:
```
c_compiler:
  - clang
c_compiler_version:
  - 20
cxx_compiler:
  - clangxx
cxx_compiler_version:
  - 20
```